### PR TITLE
Enhance readme documentation

### PR DESCRIPTION
**Pull request by otaviojava**
# Changes

- Update readme file

### CODE REVIEW

**Comment by otaviojava** [comment]
@amoscatelli and @dearrudam could you review this, please?

File: README.md [hunk 1]
```diff
@@ -60,11 +74,11 @@ Add the dependency to the target project:
 </dependency>
 ```
 
-Please refer to the [AranboDB JNoSQL driver](https://github.com/eclipse/jnosql-databases#arangodb) for specific configuration.
+For specific configuration details, please refer to the [AranboDB JNoSQL driver](https://github.com/eclipse/jnosql-databases#arangodb).
```
Comment (dearrudam) [inline README.md:77]:
```suggestion
For specific configuration details, please refer to the [ArangoDB JNoSQL driver](https://github.com/eclipse/jnosql-databases#arangodb).
```

File: README.md [hunk 2]
```diff
@@ -133,11 +154,11 @@ Add the dependency to the target project:
...
```
Comment (dearrudam) [inline README.md:157]:
```suggestion
For specific configuration details, please refer to the [ArangoDB JNoSQL driver](https://github.com/eclipse/jnosql-databases#arangodb).
```

File: README.md [hunk 3]
```diff
@@ -185,49 +206,56 @@ Add the dependency to the target project:
...
-</table>
+## Contributors ✨
 
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
+Thanks to these wonderful people for their contributions:
 
-<!-- ALL-CONTRIBUTORS-LIST:END -->
+- [amoscatelli](https://github.com/amoscatelli) 💻 🚧
+- [dearrudam](https://github.com/dearrudam) 💻 🚧
```
Comment (dearrudam) [inline README.md:259]:
```suggestion
- [dearrudam](https://github.com/dearrudam) 💻 🚧
- [otaviojava](https://github.com/otaviojava) 💻 🚧
```

**Review by dearrudam — CHANGES_REQUESTED** [review]
@otaviojava, thank you so much for this documentation enhancement! I've suggested some changes also! See ya!!!

**Review by dearrudam — APPROVED** [review]

**Comment by otaviojava** [comment]
@gastaldi could you review/merge this one?

File: README.md [hunk 4]
```diff
@@ -5,42 +5,54 @@
 
 [![Version](https://img.shields.io/maven-central/v/io.quarkiverse.jnosql/quarkus-jnosql-core?logo=apache-maven&style=flat-square)](https://search.maven.org/artifact/io.quarkiverse.jnosql/quarkus-jnosql-core)
 
-The Quarkus JNoSql Extension adds support for 
-[JNoSQL](http://www.jnosql.org/), an implementation of [Jakarta NoSQL](https://jakarta.ee/specifications/nosql/).
+This documentation provides instructions on how to integrate JNoSQL, an implementation of [Jakarta NoSQL](https://jakarta.ee/specifications/nosql/), into a Quarkus project using the Quarkus JNoSQL Extension. This extension supports JNoSQL and facilitates using NoSQL databases in your Quarkus applications.
```
Comment (gastaldi) [inline README.md:8]: Why removing the link to https://www.jnosql.org/?

**Comment by gastaldi** [comment]
Although it looks okay, I'd prefer to have this information in https://github.com/quarkiverse/quarkus-jnosql/blob/master/docs/modules/ROOT/pages/index.adoc instead (which is published in https://docs.quarkiverse.io/quarkus-jnosql/dev/index.html consequently).

**Comment by otaviojava** [comment]
> https://docs.quarkiverse.io/quarkus-jnosql/dev/index.html

I will update this one as well. Thank you.

File: README.md [hunk 5]
```diff
@@ -185,49 +206,57 @@ Add the dependency to the target project:
...
+## Contributors ✨
 
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
+Thanks to these wonderful people for their contributions:
 
-<!-- ALL-CONTRIBUTORS-LIST:END -->
+- [amoscatelli](https://github.com/amoscatelli) 💻 🚧
+- [dearrudam](https://github.com/dearrudam) 💻 🚧
+- [otaviojava](https://github.com/otaviojava) 💻 🚧
```
Comment (gastaldi) [inline README.md:260]: This section should be only changed by the @all-contributors bot (maintainers should trigger it by commenting something like `@all-contributors add @otaviojava for code and maintenance`

**Comment by otaviojava** [comment]
> Although it looks okay, I'd prefer to have this information in https://github.com/quarkiverse/quarkus-jnosql/blob/master/docs/modules/ROOT/pages/index.adoc instead (which is published in https://docs.quarkiverse.io/quarkus-jnosql/dev/index.html consequently).

I updated both; I will include more documentation at this index as next step.

**Review by dearrudam — APPROVED** [review]

**Comment by amoscatelli** [comment]
LGTM thank you all

**Pull request merged**